pkgs/development/libraries/pipewire/pipewire-pulse-path.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/meson_options.txt b/meson_options.txt
+index 4b9e46b8..9d73ed06 100644
+--- a/meson_options.txt
++++ b/meson_options.txt
+@@ -147,3 +147,6 @@ option('pw-cat',
+ option('udevrulesdir',
+        type : 'string',
+        description : 'Directory for udev rules (defaults to /lib/udev/rules.d)')
++option('pipewire_pulse_prefix',
++       type : 'string',
++       description : 'Install directory for the pipewire-pulse daemon')
+diff --git a/src/daemon/systemd/user/meson.build b/src/daemon/systemd/user/meson.build
+index 29fc93d4..4161f797 100644
+--- a/src/daemon/systemd/user/meson.build
++++ b/src/daemon/systemd/user/meson.build
+@@ -6,7 +6,8 @@ install_data(
+ 
+ systemd_config = configuration_data()
+ systemd_config.set('PW_BINARY', join_paths(pipewire_bindir, 'pipewire'))
+-systemd_config.set('PW_PULSE_BINARY', join_paths(pipewire_bindir, 'pipewire-pulse'))
++pipewire_pulse_bindir = join_paths(get_option('pipewire_pulse_prefix'), get_option('bindir'))
```

Comment:
`bindir` might become an absolute path if we add `bin` output, in which case, it would point to that input instead. Using `bin` directly will be safer.